src/transformers/testing_utils.py

```diff
@@ -1192,6 +1193,13 @@ def require_cython(test_case):
     return unittest.skipUnless(is_cython_available(), "test requires cython")(test_case)
 
 
+def require_tiktoken(test_case):
+    """
+    Decorator marking a test that requires TikToken. These tests are skipped when TikToken isn't installed.
+    """
+    return unittest.skipUnless(is_tiktoken_available(), "test requires TikToken")(test_case)
```

Comment:
do we only need tiktoken ? (I think blobfile should be added here as our tiktoken thing needs both